src/c/user-subscription-box.js

```diff
@@ -0,0 +1,153 @@
+import m from 'mithril';
+import _ from 'underscore';
+import I18n from 'i18n-js';
+import h from '../h';
+import moment from 'moment';
+import models from '../models';
+import {catarse} from '../api';
+import contributionVM from '../vms/contribution-vm';
+import ownerMessageContent from '../c/owner-message-content';
+import modalBox from '../c/modal-box';
+import userVM from '../vms/user-vm';
+
+const I18nScope = _.partial(h.i18nScope, 'payment.state');
+const contributionScope = _.partial(h.i18nScope, 'users.contribution_row');
+
+const userSubscriptionBox = {
+    controller(args) {
+        const subscription=args.subscription,
+            displayModal = h.toggleProp(false, true),
+            contactModalInfo = m.prop({});
+
+        const filterProjVM = catarse.filtersVM({
+                project_id: 'eq'
+            }).project_id(subscription.project_external_id),
+            lProj = catarse.loaderWithToken(models.project.getRowOptions(filterProjVM.parameters()));
+        
+        lProj.load().then(function(arr){
+            subscription.project = arr[0];
+            contactModalInfo({
+                id: subscription.project.project_user_id,
+                name: subscription.project.owner_name,
+                project_id: subscription.project.project_id
+            });
+            // console.log('subscription1:', JSON.stringify(subscription, null, 2));
+        });
+
+        if (subscription.reward_external_id) {
+            const filterRewVM = catarse.filtersVM({
+                    id: 'eq'
+                }).id(subscription.reward_external_id),
+                lRew = catarse.loaderWithToken(models.rewardDetail.getRowOptions(filterRewVM.parameters()));
```

Comment:
Can't we use the new rewards endpoint here? https://comum.gitbooks.io/docs/content/project/list-rewards.html

Reply:
I think that, not yet